### PR TITLE
Aw 11 build the item page

### PR DESCRIPTION
**Pull request by JosephChuksN**
Item page

### CODE REVIEW

File: src/components/Items/Item.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        </span>
+        <span>
+          <h1>ITEM NAME</h1>
+        </span>
+        <span className="flex gap-2 items-center ">
+          <h2 className="text-base text-black font-medium ">Price:</h2>
+          <h2 className="text-sm text-black font-medium">$100</h2>
+        </span>
+        <span className="flex gap-3 ">
+          <img src={shirt} alt="shirt img" className="w-12 h-12" />
```
Comment (eimaam) [inline src/components/Items/Item.tsx:25]: Can we have the cursor on this be a pointer on hover? To tell User they can be clicked for better user experience.

File: src/components/Items/RelatedItems.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import shirt from '../Home/assets/shirt.jpg'
+
+const RelatedItems:React.FC = () => {
+  return (
+    <div className="w-full flex flex-col gap-3">
+      <h2 className="text-black font-medium text-2xl">Related Porducts</h2>
```
Comment (eimaam) [inline src/components/Items/RelatedItems.tsx:7]: Just typographical error here on the _Products_...

**Review by eimaam — CHANGES_REQUESTED** [review]
Looks good to me. 
Dropped some comments. Review and merge. 🚀

File: src/components/Items/Item.tsx (hunk 1 above)
Comment (JosephChuksN) [reply src/components/Items/Item.tsx:25]: Ok! Got it

**Review by eimaam — APPROVED** [review]

**Pull request merged**